README.md

```diff
@@ -221,7 +221,7 @@ The both defaults badge (noob) already was added to all users that we already ha
 Inside your application if you want to give 100 points to some user, inside your function you have to use the following method:
 
 ```ruby
-kind =  Kind.where(:name => "teacher")
```

Comment:
LOL `the red - indicates it's a removal` yup I know it seems I was pretty sleepy when reviewing it haha sorry, just pointed out my own error 😁